content/en/agent/remote_config/_index.md

```diff
@@ -108,13 +108,18 @@ To enable Remote Configuration:
 2. Ensure your RBAC permissions include [`api_keys_write`][5], so you can create a new API key with the Remote Configuration capability, or add the capability to an existing API key. Contact your organization's Datadog administrator to update your permissions if you don't have it. A key with this capability allows you to authenticate and authorize your Agent to use Remote Configuration.
 
 3. On the [Remote Configuration][8] page, enable Remote Configuration. This enables Datadog components across your organization to receive configurations from Datadog.
+**Note:** Beginning April 8, 2024, Remote Configuration is on-by-default for:
+* New sub-organizations created by existing Datadog customers who already have enabled RC at the master organization level and where they are creating new sub-organizations in the same Datadog site as their master organization.
+* Organizations created by new Datadog customers.
+  
+To opt-out of Remote Configuration use, see the [opt-out section][23].
 
-4. Select an existing API key or create a new API key, and enable the Remote Configuration capability on the key:
+6. Select an existing API key or create a new API key, and enable the Remote Configuration capability on the key. If you match the new organization conditions mentioned in step 3, your API keys will be Remote Configuration- capable by default.
```

Comment:
```suggestion
6. Select an existing API key or create a new API key, and enable the Remote Configuration capability on the key. If your new organization fulfills the conditions mentioned in step 3, Remote Configuration is enabled on your API keys be default.
```

How exactly do you enable Remote Configuration on a new or existing API key? Even if it seems obvious, we should explicitly say something like "go to the API keys page and use the toggle"

Reply:
Good Q. The enabling API keys for RC is done through RC onboarding page itself (it's the 2nd step in RC onboarding). Users don't have to go to the API keys page.